services/workflows-service/src/collection-flow/collection-flow.service.ts

```diff
@@ -40,10 +42,13 @@ export class CollectionFlowService {
         workflowDefinitionId: credentials.flowType,
         context: {
           entity: {
-            endUserId: newUser.id,
             ballerineEntityId: newUser.businesses.at(-1)?.id,
             type: 'business',
-            data: {},
+            data: {
+              additionalInformation: {
+                endUserId: newUser.id,
```

Comment:
Note `entity` may hold both `id` and `ballerineEntityId`. Is there a specific need for `endUserId` and for it to be somewhere specific?